src/requirements/Github/components/MinMaxAmountFormControls.tsx

```diff
@@ -1,54 +1,99 @@
 import { FormControl, FormErrorMessage, FormLabel } from "@chakra-ui/react"
+import { ControlledRelativeTimeInput } from "components/common/RelativeTimeInput"
 import { ControlledTimestampInput } from "components/common/TimestampInput"
 import { useFormState, useWatch } from "react-hook-form"
 import parseFromObject from "utils/parseFromObject"
 
 type Props = {
   baseFieldPath: string
-  minAmountLabel: string
-  maxAmountLabel: string
+  minAmountLabel?: string
+  maxAmountLabel?: string
+  type?: string
+  timestamp?: boolean
 }
 
 const MinMaxAmountFormControls = ({
```

Comment:
I think we could do a great refactor here actually!

My idea is to completely get rid of the `MinMaxAmountFormControls` component (since it was GitHub specific before & now when someone is reading our code, it is not obvious at first glance that these are actually time inputs), and make 2 different components inside `src/requirements/common`:
- `RelativeMinMaxTimeFormControls` - contains only the absolute time picker logic
- `AbsoluteMinMaxTimeFormControls` - contains only the relative time picker logic
- both of these could expose a `minTimeLabel` & `maxTimeLabel` prop, and we shouldn't use the `type`, `timestamp` & `baseFieldPath` props, instead we could just pass `minTimeFieldName` & `maxTimeFieldName` (strings) and then use those instead of doing these `timestamp ? "timestamps." : ""` conditions everywhere (for errors, if I'm not missing anything, we could just use `parseFromObject(errors, minTimeFieldName)`) ;)

This way, these components would be completely reusable, even if the field paths change in the future, or if we'll need to use them for completely different fields.

Let me know what do you think about this idea, or if you think I missed something!